stubs/test.unit.stub

```diff
@@ -1,16 +0,0 @@
-<?php
-
-namespace {{ namespace }};
-
-use Tests\TestCase;
-
-class {{ class }} extends TestCase
```

Comment:
I added this originally because the default  test case imports from the PHPUnit test case not our custom one.

Reply:
I should've put a docstring to that effect on reflection

Reply:
7de5c825babd642ab9cb07758a299dfab2b71bb5